### ISSUE TITLE
Some replays don't work (replay menu doesn't appear/oos error)

### ISSUE THREAD

**Issue by Konrad22** [wesnoth/wesnoth#4004]
My replay from TLU Sc0 (Badass) and both mine and @nemaara 's replay from AtS3 S1 don't work for some reason. (The oos error doesn't happen for me, only for nemaara, no idea why.)
Since it happens in two unrelated add-ons, it might be a wesnoth engine bug.

[https://github.com/project-ethea/After_the_Storm/issues/68]

[AtS-3-Beyond her Smile replay.gz](https://github.com/wesnoth/wesnoth/files/3018741/AtS-3-Beyond.her.Smile.replay.gz)
[TLU-Morning in al-Kamija Aufzeichnung abspielen.gz](https://github.com/wesnoth/wesnoth/files/3018742/TLU-Morning.in.al-Kamija.Aufzeichnung.abspielen.gz)

[https://forums.wesnoth.org/viewtopic.php?p=639284#p639284]

**Comment by gfgtdf** [comment]
https://github.com/wesnoth/wesnoth/blob/1.14.6/src/replay_controller.cpp#L77

https://github.com/wesnoth/wesnoth/blob/1.14.6/src/display.cpp#L279

**Comment by stevecotton** [comment]
Both of these replays use `[change_theme]`, and it seems there's no trigger for replay_controller to attach itself to the new theme's theme_reset_event.

**Comment by gfgtdf** [comment]
Yes that's what I was trying to say I. My previous post, I think the easiest way to fix it would be just, in the line that recreates.tye theme to move over all event handles from the old theme object to the new theme object

**Comment by shikadiqueen** [comment]
Another possibility would be to simply ignore `[change_theme]` actions when in replay mode (no idea what should be done for multiplayer).
